pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/PersistentTopicsTest.java

```diff
@@ -232,7 +232,6 @@ public void testTerminatePartitionedTopic() {
 
     @Test
     public void testNonPartitionedTopics() {
-        pulsar.getConfiguration().setAllowAutoTopicCreation(false);
```

Comment:
This test fails if `allowAutoTopicCreation` is false.

We are testing if the subscription creation is successful here. However, if `allowAutoTopicCreation` is false, the topic cannot be created and the subscription creation should also fail.
https://github.com/apache/pulsar/blob/8061c547327b46700c8e66f2d829e258f21a292c/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java#L1787-L1789

I think it's a mistake to set `allowAutoTopicCreation` to false here.